src/promptflow/tests/sdk_cli_test/e2etests/test_cli_perf.py

```diff
@@ -19,14 +20,16 @@
 
 def run_cli_command(cmd, time_limit=3600, result_queue=None):
     from promptflow._cli._pf.entry import main
+
     sys.argv = list(cmd)
     output = io.StringIO()
 
     st = timeit.default_timer()
     with contextlib.redirect_stdout(output), mock.patch.object(
```

Comment:
@Stephen1993 please help check if this change will work

Reply:
Please help to add:
`user_agent = get_client_user_agent()`
`assert user_agent == f"{CLI_USER_AGENT} perf_monitor/1.0"`

Reply:
Updated